azurelinuxagent/common/event.py

```diff
@@ -53,15 +57,13 @@
 
 MAX_NUMBER_OF_EVENTS = 1000
 
+EVENT_FILE_EXTENSION = '.waagent.tld'
```

Comment:
Events produced by >= 2.2.47 will be saved as *.waagent.tld

Reply:
NIT: Rename this to `AGENT_EVENT_FILE_EXTENSION`

Reply:
yup, thanks!